x-pack/plugins/security_solution/public/management/pages/policy/store/policy_list/action.ts

```diff
@@ -6,6 +6,7 @@
 
 import { PolicyData } from '../../../../../../common/endpoint/types';
 import { ServerApiError } from '../../../../../common/types';
+import { GetAgentStatusResponse } from '../../../../../../../ingest_manager/common/types/rest_spec';
```

Comment:
is this exposed at top level `/common`? if so, use that path instead.
You can do this in a subsequent PR 😃